graph/pull_v2.go

```diff
@@ -211,12 +215,22 @@ func (p *v2Puller) pullV2Tag(out io.Writer, tag, taggedName string) (verified bo
 		}
 	}()
 
+	var totalSize int64
+	totalSize = 0
+	nbLayers := 0
+
+	if dryRun {
+		fmt.Printf("**** Dry Run - nothing will be downloaded ****\n")
```

Comment:
This output needs to be directed to `p.config.OutStream`, using `p.sf.FormatStatus`, instead of printing to the daemon's stdout.